packages/access-client/src/agent-data.js

```diff
@@ -111,12 +111,15 @@ export class AgentData {
 
   /**
    * @deprecated
-   * @param {import('@ucanto/interface').DID} did
+   * @param {import('@ucanto/interface').DID<'key'>} did
```

Comment:
It could be useful to indicate *which* did (`space`). (e.g. it might be ambiguous considering the `proof` @param itself is composed of several DIDs)